examples/cypress/package.json

```diff
@@ -11,9 +11,10 @@
         "@bazel/typescript": "^2.3.0",
         "@types/node": "14.0.13",
         "cypress": "^4.8.0",
-        "typescript": "3.9.5"
+        "typescript": "3.9.5",
+        "tar": "6.0.5"
```

Comment:
the user shouldn't have to depend on `tar` right? if the cypress code requires it, should come from its own dependency

Reply:
Done